model/mediafile.go

```diff
@@ -39,6 +39,7 @@ type MediaFile struct {
 	Compilation          bool      `json:"compilation"`
 	Comment              string    `json:"comment"`
 	Lyrics               string    `json:"lyrics"`
+	Bpm                  int       `json:"bpm"`
```

Comment:
I think we should tag the field as `json:"bpm,omitempty"`. This way we don't need `<FunctionField source="bpm" render={(r) => r.bpm || ''} />` as it will be empty, not zero, simplifying the Datagrids (just need a simple `<TextField/>` field

Reply:
Good call, much cleaner.